core/controllers/certificate_assessment.py

```
@@ -18,9 +18,10 @@
 
 from core import feconf, utils
 from core.controllers import acl_decorators, base
+from core.controllers import domain_objects_validator as validation_method
 from core.domain import certificate_assessment_services
 
-from typing import Dict, List, TypedDict
+from typing import Any, Dict, List, TypedDict
```

Ruff (0.16.1)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)